cockroachcloud/migrations-page.md

```diff
@@ -139,64 +139,88 @@ If the credentials are valid, they will be added to the [**Credentials** table](
-<li><b>Not Executed</b> represents a statement that was not executed by the tool, such as an <code>INSERT</code> or <code>COPY</code> statement.</li>
-<li><b>Missing User</b> represents a statement that references a nonexistent user. </li>
+**Required Fixes** indicate errors that must be resolved before you can [migrate the schema](#migrate-the-schema).
+
+|        Column       |                                                                                                                                                                                                                                                                                                             Description                                                                                                                                                                                                                                                                                                              |
+|---------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Description         | A summary of the error type.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                         |
+| Type                | The category of error:<ul><li>**Unimplemented Feature:** A statement that uses an [unimplemented feature](../{{version_prefix}}migration-overview.html#unimplemented-features).</li><li>**Missing User:** A statement that references a nonexistent user.</li><li>**Translation Failure:** (Non-PostgreSQL dialects) A statement that could not be converted because it has no equivalent syntax on CockroachDB.</li><li>**Uncategorized:** A statement that the tool did not or could not execute.</li><li>**Incidental:** A statement that failed because another SQL statement encountered one of the preceding error types.</ul> |
```

Comment:
`Type` only applies to the error fixes, not the compat notes or suggestions tables (since those would be redundant)

Reply:
Ignore, I see you've covered it below.